target/linux/octeon/image/Makefile

```diff
@@ -29,6 +29,15 @@ define Device/generic
 endef
 TARGET_DEVICES += generic
 
+ITUSROUTER_CMDLINE:=console=ttyS0,115200, root=/dev/mmcblk1p2 rootfstype=squashfs,ext4,f2fs rootwait
+define Device/itus_shield-router
+  DEVICE_VENDOR := Itus Networks
+  DEVICE_MODEL := Shield Router
+  CMDLINE := $(ITUSROUTER_CMDLINE)
```

Comment:
I think this abstraction predates the times before CMDLINE was initialized. Won't passing here the values simply work, instead of passing the variable?

Reply:
It follows the format of the existing entries, which was a big thing @adschm was requiring format-wise. I define ITUSROUTER_CMDLINE just before the define, but passing it directly shouldn't matter one way or another.

Reply:
Passing directly is cleaner IMHO, but if that was requested differently, then that's fine with me.

Reply:
I don't remember that discussion, but I don't really care here. So, as there appears to be no strong reason to change the existing "style", I'd just follow that.